WordPress/src/main/java/org/wordpress/android/ui/ExpandableHeightGridView.java

```diff
@@ -7,53 +7,43 @@
 
 // based on http://stackoverflow.com/questions/8481844/gridview-height-gets-cut
 
-public class ExpandableHeightGridView extends GridView
-{
-    boolean expanded = false;
+public class ExpandableHeightGridView extends GridView {
+    boolean mIsExpanded = false;
```

Comment:
Should the "is" be included in a variable name?

Reply:
IIRC, we added the `is`, `has` prefixes in variable and field names in FluxC to make them explicit. @aforcier do you remember if had another rationale?

Reply:
Don't think there was another rationale - it's explicit and can eliminate some ambiguities (is `mPage` an object representing a page? or is it saying that the current object is a page? etc.). I don't think we actually have an enforced rule for this though.